pkg/pipelines/utils.go

```diff
@@ -58,6 +58,13 @@ func BootstrapRepository(o *BootstrapOptions, f clientFactory, e executor) error
 	}
 	created, _, err := client.Repositories.Create(context.Background(), ri)
```

Comment:
I see that the "Create" returns (*scm.Repository, *scm.Response, error). I was thinking if the "*scm.Response" returns such error "Repo already exists"? We are ignoring "*scm.Response" in this line. Can you verify this?

Reply:
Thanks @shubhamagarwal19 .